packages/docusaurus-theme-classic/src/theme/BlogSidebar/Desktop/styles.module.css

```diff
@@ -29,6 +29,7 @@
 
 .sidebarItemLink {
   color: var(--ifm-font-color-base);
+  display: block;
```

Comment:
Just for the sake of uniformity -- since all the links in such blocks (sidebars, toc) are full-width.